lms/exceptions.py

```diff
@@ -13,7 +13,19 @@ class LTILaunchError(HTTPBadRequest):  # pylint: disable=too-many-ancestors
     pass
 
 
+class LTIContentItemError(HTTPBadRequest):  # pylint: disable=too-many-ancestors
+    """Base exception for problems handling LTI content item selection."""
+
+    pass
+
+
 class MissingLTILaunchParamError(LTILaunchError):  # pylint: disable=too-many-ancestors
     """Exception raised if params required for lti launch are missing."""
```

Comment:
This is trivial, but it would be nice for our use of "LTI" to be consistently upper-cased (as it is in other docstrings).